spec/features/sp_cost_tracking_spec.rb

```diff
@@ -76,56 +65,6 @@
     end
   end
 
-  it 'logs the correct costs for an ial1 authentication' do
-    create_ial1_user_from_sp(email)
-    SpCost.delete_all
-
-    # track costs without dealing with 'remember device'
-    Capybara.reset_session!
-
-    visit_idp_from_sp_with_ial1(:oidc)
-    fill_in_credentials_and_submit(email, password)
-    fill_in_code_with_last_phone_otp
-    click_submit_default
-
-    expect_sp_cost_type(0, 1, 'digest')
-    expect_sp_cost_type(1, 1, 'sms')
-    expect_sp_cost_type(2, 1, 'authentication')
-  end
-
-  it 'logs the correct costs for an ial2 authentication' do
-    create_ial2_user_from_sp(email)
-    SpCost.delete_all
-
-    # track costs without dealing with 'remember device'
-    Capybara.reset_session!
-
-    visit_idp_from_sp_with_ial2(:oidc)
-    fill_in_credentials_and_submit(email, password)
-    fill_in_code_with_last_phone_otp
-    click_submit_default
-
-    expect_sp_cost_type(0, 2, 'digest')
-    expect_sp_cost_type(1, 2, 'sms')
-    expect_sp_cost_type(2, 2, 'authentication')
-  end
-
-  it 'logs the correct costs for a direct authentication' do
-    visit root_path
-    create_ial1_user_directly(email)
-    SpCost.delete_all
-
-    # track costs without dealing with 'remember device'
-    Capybara.reset_session!
-
-    visit root_path
-    fill_in_credentials_and_submit(email, password)
-    fill_in_code_with_last_phone_otp
-    click_submit_default
-
-    expect_direct_cost_type(0, 'digest')
```

Comment:
Looks like this method is unused after this removal.

Reply:
Whoops, thanks for catching that: https://github.com/18F/identity-idp/pull/6266